javascript/node/selenium-webdriver/test/chrome/devtools_test.js

```diff
@@ -115,6 +115,22 @@ test.suite(
       })
     })
 
+    describe('JS DOM events', function() {
+      it('calls the event listener on dom mutations', async function() {
+        const cdpConnection = await driver.createCDPConnection('page')
+
+        await driver.logMutationEvents(cdpConnection, function(event) {
+          console.log(event)
```

Comment:
we should probably assert and not just log the item

Reply:
@AutomatedTester - This is a draft, I tried changing PR to draft, could not find an option, this isnt complete yet, I am working on it with @Harsha509